### PR TITLE
added docs/README.md

### PR DESCRIPTION
**Pull request by Brijeshthummar02**
**PR Checklist**

- [ ] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

Added file to docs/README.md
issue #550

### CODE REVIEW

**Comment by Brijeshthummar02** [comment]
@dennyglee have a look.

**Comment by HorizonNet** [comment]
Failing `Docs Generation Tests` will be resolved with the changes proposed in #646.

**Review by dennyglee — APPROVED** [review]
LGTM overall but just noticed the conflicts @Brijeshthummar02 - could you take a quick Look at this as well as address the DCO issue?  Thanks!

**Review by dennyglee — CHANGES_REQUESTED** [review]
Per readme conflict

File: mkdocs.yml [hunk 1]
```diff
@@ -22,17 +22,19 @@ nav:
...
       - Apache Spark: integrations/unity-catalog-spark.md  
       - Daft: integrations/unity-catalog-daft.md
       - DuckDB: integrations/unity-catalog-duckdb.md
       - PuppyGraph: integrations/unity-catalog-puppygraph.md
       - SpiceAI: integrations/unity-catalog-spiceai.md
       - XTable: integrations/unity-catalog-xtable.md
+      - Unity Catalog Celerdata: integrations/unity-catalog-celerdata.md
   - Deployment: deployment.md
 
-exclude_docs: |
```
Comment (HorizonNet) [inline mkdocs.yml:34]: @Brijeshthummar02 Please leave as is. This definitely works and with the update the doc test is failing again.

**Pull request closed without merging**